blueprint/src/sections/f_divergence.tex

```diff
@@ -353,6 +353,19 @@ \section{Conditional f-divergence}
   if $x \mapsto D_f(\kappa(x), \eta(x))$ is $\mu$-integrable and $+\infty$ otherwise.
 \end{definition}
 
+\begin{lemma}
+  \label{lem:condFDiv_nonneg}
+  \lean{ProbabilityTheory.condFDiv_nonneg}
+  \leanok
+  \uses{def:condFDiv}
+  Let $f : \mathbb{R} \to \mathbb{R}$ be continuous in $0$, $\mu$ a measure on $\mathcal X$ and $\kappa, \eta : \mathcal X \rightsquigarrow \mathcal Y$ two Markov kernels. If $f(1) = 0$ then $D_f(\kappa, \eta \mid \mu) \ge 0$.
```

Comment:
For now I have omitted the properties of `f` from the lemmas (they are written at the beginning of the chapter IIRC).
```suggestion
  Let $\mu$ be a measure on $\mathcal X$ and $\kappa, \eta : \mathcal X \rightsquigarrow \mathcal Y$ two Markov kernels. If $f(1) = 0$ then $D_f(\kappa, \eta \mid \mu) \ge 0$.
```